### PR TITLE
Netstats docker and node vulnerabilities fix

### PR DESCRIPTION
**Pull request by yourheropaul**
This is a small patch to resolve the Docker build fix and the package.json vulnerabilities

### CODE REVIEW

File: netstats/bin/www [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+var debug = require('debug')('eth-netstats');
```
[the remark this answers is not in the document]
Comment (yourheropaul) [reply netstats/bin/www]: It's the script that runs the application. For some reason (probably related to running it on elastic beanstalk) they've decided to implement it this way.

[the remark this answers is not in the document]
Comment (yourheropaul) [reply netstats/bin/www]: Without it the docker build process fails

[the remark this answers is not in the document]
Comment (acroca) [reply netstats/bin/www]: I mean the `debug`, doesn't seem to be used.

[the remark this answers is not in the document]
Comment (yourheropaul) [reply netstats/bin/www]: Good point, it doesn't do anything (except allow debugging in development, which is unnecessary). I've removed it.

**Pull request merged**